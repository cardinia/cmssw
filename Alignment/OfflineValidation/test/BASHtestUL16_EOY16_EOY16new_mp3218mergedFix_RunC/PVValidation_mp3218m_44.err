27-Sep-2019 21:51:51 CEST  Initiating request to open file root://eoscms.cern.ch//eos/cms/store/data/Run2016C/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/80000/4A1EDF1A-2709-E911-AE8E-1866DA85DB6B.root
27-Sep-2019 21:51:55 CEST  Successfully opened file root://eoscms.cern.ch//eos/cms/store/data/Run2016C/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/80000/4A1EDF1A-2709-E911-AE8E-1866DA85DB6B.root
Begin processing the 1st record. Run 275931, Event 1129490, LumiSection 1 on stream 0 at 27-Sep-2019 21:54:01.993 CEST
Begin processing the 1001st record. Run 275931, Event 7197905, LumiSection 5 on stream 0 at 27-Sep-2019 21:56:30.571 CEST
%MSG-w OutOfBounds:  TrackRefitter:FinalTrackRefitter 27-Sep-2019 21:57:49 CEST  Run: 275931 Event: 12716124
Prob Q outside the bounds of the quality word. Defaulting to Prob=0. Prob = 1 QualityWord = 16383
%MSG
Begin processing the 2001st record. Run 275931, Event 14822308, LumiSection 10 on stream 0 at 27-Sep-2019 21:58:50.357 CEST
%MSG-w OutOfBounds:  TrackRefitter:FinalTrackRefitter 27-Sep-2019 22:00:09 CEST  Run: 275931 Event: 19940419
Prob Q outside the bounds of the quality word. Defaulting to Prob=0. Prob = 1 QualityWord = 1151
%MSG
Begin processing the 3001st record. Run 275931, Event 23962402, LumiSection 15 on stream 0 at 27-Sep-2019 22:01:03.246 CEST
Begin processing the 4001st record. Run 275931, Event 31004346, LumiSection 20 on stream 0 at 27-Sep-2019 22:03:11.275 CEST
Begin processing the 5001st record. Run 275931, Event 37577686, LumiSection 24 on stream 0 at 27-Sep-2019 22:05:19.373 CEST
Begin processing the 6001st record. Run 275931, Event 47159427, LumiSection 29 on stream 0 at 27-Sep-2019 22:07:32.230 CEST
Begin processing the 7001st record. Run 275931, Event 54080029, LumiSection 34 on stream 0 at 27-Sep-2019 22:09:45.977 CEST
Begin processing the 8001st record. Run 275931, Event 62184462, LumiSection 39 on stream 0 at 27-Sep-2019 22:12:00.469 CEST
%MSG-w OutOfBounds:  TrackRefitter:FinalTrackRefitter 27-Sep-2019 22:12:05 CEST  Run: 275931 Event: 62248124
Prob Q outside the bounds of the quality word. Defaulting to Prob=0. Prob = 1 QualityWord = 935
%MSG
Begin processing the 9001st record. Run 275931, Event 71119673, LumiSection 44 on stream 0 at 27-Sep-2019 22:14:14.904 CEST
%MSG-w OutOfBounds:  TrackRefitter:FinalTrackRefitter 27-Sep-2019 22:14:38 CEST  Run: 275931 Event: 72186495
Prob Q outside the bounds of the quality word. Defaulting to Prob=0. Prob = 1 QualityWord = 4
%MSG
%MSG-w OutOfBounds:  TrackRefitter:FinalTrackRefitter 27-Sep-2019 22:15:21 CEST  Run: 275931 Event: 74442756
Prob Q outside the bounds of the quality word. Defaulting to Prob=0. Prob = 1 QualityWord = 1754
%MSG
Begin processing the 10001st record. Run 275931, Event 77349152, LumiSection 48 on stream 0 at 27-Sep-2019 22:16:26.614 CEST
Begin processing the 11001st record. Run 275931, Event 85233805, LumiSection 53 on stream 0 at 27-Sep-2019 22:18:33.696 CEST
Begin processing the 12001st record. Run 275931, Event 93610600, LumiSection 58 on stream 0 at 27-Sep-2019 22:20:46.068 CEST
Begin processing the 13001st record. Run 275931, Event 100858138, LumiSection 63 on stream 0 at 27-Sep-2019 22:23:01.634 CEST
Begin processing the 14001st record. Run 275931, Event 109469124, LumiSection 68 on stream 0 at 27-Sep-2019 22:25:15.409 CEST
Begin processing the 15001st record. Run 275931, Event 115625313, LumiSection 72 on stream 0 at 27-Sep-2019 22:27:31.706 CEST
Begin processing the 16001st record. Run 275931, Event 123291662, LumiSection 77 on stream 0 at 27-Sep-2019 22:29:51.315 CEST
Begin processing the 17001st record. Run 275931, Event 131409222, LumiSection 82 on stream 0 at 27-Sep-2019 22:32:08.784 CEST
Begin processing the 18001st record. Run 275931, Event 139549441, LumiSection 87 on stream 0 at 27-Sep-2019 22:34:24.218 CEST
27-Sep-2019 22:35:41 CEST  Closed file root://eoscms.cern.ch//eos/cms/store/data/Run2016C/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/80000/4A1EDF1A-2709-E911-AE8E-1866DA85DB6B.root

=============================================

MessageLogger Summary

 type     category        sev    module        subroutine        count    total
 ---- -------------------- -- ---------------- ----------------  -----    -----
    1 OutOfBounds          -w TrackRefitter:Fi                       5        5
    2 fileAction           -s file_close                             1        1
    3 fileAction           -s file_open                              2        2

 type    category    Examples: run/evt        run/evt          run/evt
 ---- -------------------- ---------------- ---------------- ----------------
    1 OutOfBounds          275931/12716124  275931/19940419  275931/74442756
    2 fileAction           PostGlobalEndRun                  
    3 fileAction           pre-events       pre-events       

Severity    # Occurrences   Total Occurrences
--------    -------------   -----------------
Warning                 5                   5
System                  3                   3

dropped waiting message count 0
[8.944MB/8.944MB][100%][==================================================][8.944MB/s]  [8.944MB/8.944MB][100%][==================================================][8.944MB/s]  
