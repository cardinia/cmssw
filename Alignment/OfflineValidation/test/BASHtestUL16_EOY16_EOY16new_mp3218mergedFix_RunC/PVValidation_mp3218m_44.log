000 (3196550.044.000) 09/27 21:32:18 Job submitted from host: <137.138.44.75:9618?addrs=137.138.44.75-9618+[2001-1458-d00-1a--b]-9618&alias=bigbird14.cern.ch&noUDP&sock=schedd_2873_37ae_40>
...
040 (3196550.044.000) 09/27 21:42:32 Started transferring input files
	Transferring to host: <188.185.245.34:9618?addrs=188.185.245.34-9618+[2001-1459-301-88--100-7]-9618&noUDP&sock=16598_8adc_30825>
...
040 (3196550.044.000) 09/27 21:42:33 Finished transferring input files
...
001 (3196550.044.000) 09/27 21:42:33 Job executing on host: <188.185.245.34:9618?addrs=188.185.245.34-9618+[2001-1459-301-88--100-7]-9618&noUDP&sock=16491_0866_3>
...
006 (3196550.044.000) 09/27 21:42:41 Image size of job updated: 448
	1  -  MemoryUsage of job (MB)
	448  -  ResidentSetSize of job (KB)
...
006 (3196550.044.000) 09/27 21:47:41 Image size of job updated: 113928
	91  -  MemoryUsage of job (MB)
	92372  -  ResidentSetSize of job (KB)
...
006 (3196550.044.000) 09/27 21:52:41 Image size of job updated: 298700
	266  -  MemoryUsage of job (MB)
	272080  -  ResidentSetSize of job (KB)
...
006 (3196550.044.000) 09/27 21:57:42 Image size of job updated: 589120
	529  -  MemoryUsage of job (MB)
	540820  -  ResidentSetSize of job (KB)
...
006 (3196550.044.000) 09/27 22:02:43 Image size of job updated: 589256
	529  -  MemoryUsage of job (MB)
	540956  -  ResidentSetSize of job (KB)
...
006 (3196550.044.000) 09/27 22:07:43 Image size of job updated: 591240
	531  -  MemoryUsage of job (MB)
	542940  -  ResidentSetSize of job (KB)
...
006 (3196550.044.000) 09/27 22:12:43 Image size of job updated: 592188
	532  -  MemoryUsage of job (MB)
	543888  -  ResidentSetSize of job (KB)
...
006 (3196550.044.000) 09/27 22:17:44 Image size of job updated: 592392
	532  -  MemoryUsage of job (MB)
	544092  -  ResidentSetSize of job (KB)
...
006 (3196550.044.000) 09/27 22:32:45 Image size of job updated: 593248
	533  -  MemoryUsage of job (MB)
	545200  -  ResidentSetSize of job (KB)
...
040 (3196550.044.000) 09/27 22:36:00 Started transferring output files
...
040 (3196550.044.000) 09/27 22:36:00 Finished transferring output files
...
005 (3196550.044.000) 09/27 22:36:01 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 00:40:05, Sys 0 00:00:06  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:40:05, Sys 0 00:00:06  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	6286  -  Run Bytes Sent By Job
	1018  -  Run Bytes Received By Job
	6286  -  Total Bytes Sent By Job
	1018  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :                 1           
	   Disk (KB)            :     9203        1   4149941 
	   Memory (MB)          :      533     2000      2000 
...
