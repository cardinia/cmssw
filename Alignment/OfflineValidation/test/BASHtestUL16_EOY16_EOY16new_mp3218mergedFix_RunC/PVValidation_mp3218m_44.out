Job started at  Fri Sep 27 21:42:48 CEST 2019
batch dir: /pool/condor/dir_14845 release: /afs/cern.ch/cms/CAF/CMSALCA/ALCA_TRACKERALIGN/data/commonValidation/alignmentObjects/paconnor/CMSSW_10_6_0/src release base: /cvmfs/cms.cern.ch/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_0
cmsRun PVValidation_mp3218m_275931_cfg.py
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: This is DATA!
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: Applying TrackerSurfaceDeformations!
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: Running DA Algorithm!
Content of working dir is total 9.0M -rw-r--r--. 1 tkello zh 9.0M Sep 27 22:35 PVValidation_mp3218m_275931.root -rw-r--r--. 1 tkello zh 20K Sep 27 21:42 PVValidation_mp3218m_275931_cfg.py -rw-r--r--. 1 tkello zh 5.0K Sep 27 22:35 _condor_stderr -rw-r--r--. 1 tkello zh 522 Sep 27 21:45 _condor_stdout -rwxr-xr-x. 1 tkello zh 1018 Sep 27 21:42 condor_exec.exe -rw-------. 1 tkello zh 4.3K Sep 27 21:51 tkello.cc drwx------. 3 tkello zh 4.0K Sep 27 21:42 tmp drwx------. 3 tkello zh 4.0K Sep 27 21:42 var
Job ended at  Fri Sep 27 22:36:00 CEST 2019
